README.md

```diff
@@ -116,19 +118,36 @@ $ mkdir build
 $ cd build
 ```
 
-* Call cmake with the relevant options:
+* Configure with cmake and add relevant options:
 
 ```sh
-$ cmake -DCMAKE_INSTALL_PREFIX=/opt/Vc -DBUILD_TESTING=OFF <srcdir>
+$ cmake -DBUILD_TESTING=OFF ..
```

Comment:
How about making `BUILD_TESTING` off by default and adding `-DBUILD_TESTING=ON` to the CI?

Reply:
We could do that, but that could affect a lot of downstream users.

Reply:
New PR here: #325